src/components/user/user.service.ts

```diff
@@ -362,14 +362,15 @@ export class UserService {
 
     // Update email
     if (input.email) {
+      const relationshipsOfPropertiesToDelete: BaseNodePropertyRelationships<User> = [
+        'email',
+      ];
+
       // Remove old emails and relations
-      const uniqueProperties: UniqueProperties<User> = {
-        email: ['Property', 'EmailAddress'],
-      };
       await this.db
         .query()
         .match([node('node', ['User', 'BaseNode'], { id: user.id })])
-        .call(setPropLabelsAndValuesDeleted, uniqueProperties)
+        .call(deleteProperties, relationshipsOfPropertiesToDelete)
```

Comment:
I believe this is the only place we're deleting an individual property in the API so far.